extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -154,7 +154,8 @@ export class AmpAnalytics extends AMP.BaseElement {
     // Trigger callback can be synchronous. Do the registration at the end.
     for (const k in this.config_['triggers']) {
       if (this.config_['triggers'].hasOwnProperty(k)) {
-        const trigger = this.config_['triggers'][k];
+        let trigger = null;
```

Comment:
I tried a few things on their sandbox but couldn't repro. I am fairly sure some combination of flags we are using is creating ES5 code where the symbol can not be a constant and type check runs on transpiled code AFAIK.